client/components/masonry-grid/index.tsx

```diff
@@ -0,0 +1,86 @@
+import clsx from 'clsx';
+import { debounce } from 'lodash';
+import { FC, PropsWithChildren, useRef, useLayoutEffect } from 'react';
+
+import './style.scss';
+
+const calculateMasonryGrid = ( wrapper: HTMLElement ) => {
+	const wrapperStyle = getComputedStyle( wrapper );
+	const columnCount = wrapperStyle.gridTemplateColumns.split( ' ' ).length;
+	const parsedRowGap = /(\d+(\.\d+)?)px/.exec( wrapperStyle.rowGap );
```

Comment:
We talked about dropping this in favor of a plain `parseFloat` call. Curious if there was a reason for keeping it in, @nightnei?